Explorer/Assets/DCL/MapRenderer/CoordsUtils/ChunkCoordsUtils.cs

```diff
@@ -64,7 +64,21 @@ public bool TryGetCoordsWithinInteractableBounds(Vector3 pos, out Vector2Int coo
         public Vector3 CoordsToPosition(Vector2Int coords) =>
             (Vector2)(coords * ParcelSize);
 
+        public IEnumerable<Vector2Int> ConvertToVector2Int(IEnumerable<string> coordinatesList)
+        {
+            foreach (string coordinates in coordinatesList)
+            {
+                string[] parts = coordinates.Split(',');

```

Comment:
There are already several methods to parse coordinates: the one without allocations is `IpfsHelper.DecodePointer`. I propose you to unify them